website/src/components/CalendarIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import type { ComponentPropsWithoutRef } from 'react'
+
+interface CalendarIconProps extends ComponentPropsWithoutRef<'svg'> {
```

Comment:
We should extend `ExperimentalIconProps` (or `WithOptional<ExperimentalIconProps, 'alt'>`) to support the `size` and `color` (CSS) props, and wrap the `<svg>` in `<ExperimentalIcon {...props}>` to make sure it works well everywhere icons are accepted. It does a bunch of helpful things, e.g. it sets `shrink-0` which ensures the icon doesn't get squished in a flex container with wrapping text, it is properly recognized as an icon in components that need to have different styles when they only contain an icon (e.g. `ExperimentalButton`), etc.